packages/manager/apps/pci-file-storage/src/hooks/useGetProject.tsx

```diff
@@ -0,0 +1,31 @@
+import { useEffect, useState } from 'react';
+
+import { useProject } from '@ovh-ux/manager-pci-common';
+import { useNavigation } from '@ovh-ux/manager-react-shell-client';
+
+export type TProject = {
+  id: string;
+  name: string;
+  url: string;
+};
+
+export const useGetProject = () => {
+  const navigation = useNavigation();
+  const { data: project } = useProject();
```

Comment:
I think you should pick then `useProjectId` from instance in order to use react router dom instead of `pci-common`. And you'll be sure to have a `projectId` returned